ui/oopsyraidsy/data/06-ew/raid/p10n.ts

```diff
@@ -1,12 +1,118 @@
+import NetRegexes from '../../../../../resources/netregexes';
 import ZoneId from '../../../../../resources/zone_id';
 import { OopsyData } from '../../../../../types/data';
 import { OopsyTriggerSet } from '../../../../../types/oopsy';
 
-export type Data = OopsyData;
+// TODO: Replace Unmitigated Explosion by detecting/blaming whomever did not soak a pillar.
+
+export interface Data extends OopsyData {
+  bindingSoulPlayers?: string[];
+  concentratedPoisonPlayers?: string[];
+}
 
 const triggerSet: OopsyTriggerSet<Data> = {
   zoneId: ZoneId.AnabaseiosTheTenthCircle,
-  damageWarn: {},
+  damageWarn: {
+    'P10N Parted Plumes Blade': '8271', // rotating blade AoE
+    'P10N Pandaemoniac Ray': '8266', // half-room AoE
+  },
+  damageFail: {
+    'P10N Unmitigated Explosion': '8260', // unsoaked pillar
+    'P10N Imprisonment': '8262', // circle AoE from jails
+    'P10N Cannonspawn': '8264', // donut AoE from jails
+  },
+  gainsEffectWarn: {
+    'P10N Parted Plumes Puddle': 'C05', // Bleeding debuff from standing in center puddle
+  },
+  soloWarn: {
+    'P10N Pandaemoniac Meltdown': '8277', // line stack
+  },
+  triggers: [
+    {
+      // Stacking debuff that auto-increments and upon reaching 5 stacks is fatal.
+      id: 'P10N Binding Soul Share Gain',
```

Comment:
This is from Silkspit.  If you don't spread sufficiently, you can be tethered to player(s)/posts(s) and receive the 'Binding Soul Share' debuff.  Can always change the trigger name to Silkspit, although I think (unverified) that you can also take damage from standing in a Silkspit puddle, so that could be confusing.

Reply:
Makes sense! I just didn't know what this was offhand.  Could the comment say "Stacking debuff from Silkspit ... etc" to make this more clear?